src/test/java/com/mongodb/jdbc/MongoConnectionTest.java

```diff
@@ -21,7 +21,7 @@
 @ExtendWith(MockitoExtension.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @MockitoSettings(strictness = Strictness.WARN)
-class MongoConnectionTest extends MongoMock {
+class MongoConnectionTest extends MySQLMock {
```

Comment:
This is also covered in the other outstanding PR, changed just enough to fix it here (so not split into two test classes)